File: Data Structures & Algorithms/concatenation-of-array/submission-1.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = [None for i in range(2*len(nums))]
        i = 0;
        while i < len(nums):
            ans[i] = nums[i]
            ans[i+n] = nums[i]
            i+=1
        return ans

File: Data Structures & Algorithms/two-integer-sum/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const counts = {}; // number: index
        for (let i = 0; i < nums.length; i++) {
            let potentialAnswer = target - nums[i];
            if (potentialAnswer in counts) return [counts[potentialAnswer], i];
            else counts[nums[i]] = i;
        }
    }
}
